.github/dependabot.yml

```diff
@@ -51,3 +51,8 @@ updates:
       interval: "daily"
     labels:
       - "CL-BuildPackaging"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
```

Comment:
Shouldn't this be `/.github/workflows`? All GitHub actions are deployed in that folder.

Reply:
Docs say to set it to `/` for GitHub Actions ([1](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#:~:text=Set%20the%20directory%20to%20%22/%22%20to%20check%20for%20workflow%20files%20in%20.github/workflows.), [2](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#directory)).

> Set the directory to "/" to check for workflow files in .github/workflows.